doc/application_plugins/index.rst

```diff
@@ -266,16 +267,11 @@ Anyway, in a productive environment you probably want to uncheck this feature.
 OTRS
 ----
 
-There are two plugins for OTRS. For OTRS version 4.0 and higher use
-*privacyIDEA-4_0.pm*.
+The OTRS Plugin can be found in its own
+`GitHub Repository <https://github.com/privacyidea/otrs>`__.
```

Comment:
Are two underscores at the end correct?

Reply:
Yes, it's an anonymous reference (https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557). Otherwise sphinx would complain about to many "Duplicate explicit target name".